docs/install/harvester-configuration.md

```diff
@@ -441,3 +442,18 @@ install:
   vip_mode: dhcp
   vip_hw_addr: 52:54:00:ec:0e:0b
 ```
+
+### `install.no_data_partition`
+
+#### Definition
+
+Do not create an additional disk partition for storing VM data
+and use operating system partition to store them. Default: `false`.
```

Comment:
```suggestion
Do not create an additional disk partition for storing VM data. The operating system partition will be used to store the VM data. 

Default: `false`.
```

Reply:
Rephrased it a bit based on your suggestion.